plugins/field-bitmap/src/field-bitmap.js

```diff
@@ -461,6 +517,25 @@ export class FieldBitmap extends Blockly.Field {
   setPixel_(r, c, newValue) {
     const newGrid = JSON.parse(JSON.stringify(this.getValue()));
     newGrid[r][c] = newValue;
+
+    /*
+    Modify the setPixel method to fire BLOCK_FIELD_INTERMEDIATE_CHANGE
+    events. Make sure to call setValue still, but pass false for the
+    second parameter.
+    */
```

Comment:
Can you delete these comments (here and above).

Writing code comments is great! But It's more useful when they explain /why/ something is happening, rather than /what/ is happening. The code should already tell you what is happening!